### PR TITLE
NB11 Quality Control View Fix 

### PR DESCRIPTION
**Pull request by arcturus2**
### Description of the Change

I've commented out the references to the obsolete JavaFX skins which will prevent the NoClassFound errors. There is more work to be done which is documented in #308.

### Alternate Designs

Requires more investigation but we could use the ControlsFX skins. Not sure how they can be used so I felt this could be an investigative task in #308.

### CODE REVIEW

File: CoreQualityControlView/src/au/gov/asd/tac/constellation/views/qualitycontrol/resources/quality-control-view.css [hunk 1]
```diff
@@ -13,7 +13,7 @@
  ******************************************************************************/
 
 .table-view {
-    -fx-skin: "com.sun.javafx.scene.control.skin.TableViewSkin";
+    /*-fx-skin: "com.sun.javafx.scene.control.skin.TableViewSkin";*/
```
Comment (cygnus-x-1) [inline CoreQualityControlView/src/au/gov/asd/tac/constellation/views/qualitycontrol/resources/quality-control-view.css]: I believe this still exists, it's just moved - https://openjfx.io/javadoc/14/javafx.controls/javafx/scene/control/skin/TableViewSkin.html

File: CoreQualityControlView/src/au/gov/asd/tac/constellation/views/qualitycontrol/resources/quality-control-view.css [hunk 2]
```diff
@@ -351,7 +351,7 @@
  ******************************************************************************/
 
 .button {
-    -fx-skin: "com.sun.javafx.scene.control.skin.ButtonSkin";
+    /*-fx-skin: "com.sun.javafx.scene.control.skin.ButtonSkin";*/
```
Comment (cygnus-x-1) [inline CoreQualityControlView/src/au/gov/asd/tac/constellation/views/qualitycontrol/resources/quality-control-view.css]: https://openjfx.io/javadoc/14/javafx.controls/javafx/scene/control/skin/ButtonSkin.html

File: CoreQualityControlView/src/au/gov/asd/tac/constellation/views/qualitycontrol/resources/quality-control-view.css [hunk 3]
```diff
@@ -132,7 +132,7 @@
 }
 
 .table-cell {
-    -fx-skin: "com.sun.javafx.scene.control.skin.TableCellSkin";
+    /*-fx-skin: "com.sun.javafx.scene.control.skin.TableCellSkin";*/
```
Comment (cygnus-x-1) [inline CoreQualityControlView/src/au/gov/asd/tac/constellation/views/qualitycontrol/resources/quality-control-view.css]: https://openjfx.io/javadoc/14/javafx.controls/javafx/scene/control/skin/TableCellSkin.html

File: CoreQualityControlView/src/au/gov/asd/tac/constellation/views/qualitycontrol/resources/quality-control-view.css [hunk 4]
```diff
@@ -62,7 +62,7 @@
 /* Each row in the table is a table-row-cell. Inside a table-row-cell is any
    number of table-cell. */
 .table-row-cell {
-    -fx-skin: "com.sun.javafx.scene.control.skin.TableRowSkin";
+    /*-fx-skin: "com.sun.javafx.scene.control.skin.TableRowSkin";*/
```
Comment (cygnus-x-1) [inline CoreQualityControlView/src/au/gov/asd/tac/constellation/views/qualitycontrol/resources/quality-control-view.css]: https://openjfx.io/javadoc/14/javafx.controls/javafx/scene/control/skin/TableRowSkin.html

**Comment by arcturus2** [comment]
Thanks @cygnus-x-1 I was able to use them and it worked like a charm. Please review.

File: CoreQualityControlView/src/au/gov/asd/tac/constellation/views/qualitycontrol/resources/quality-control-view.css (hunk 1 above)
Comment (arcturus2) [reply CoreQualityControlView/src/au/gov/asd/tac/constellation/views/qualitycontrol/resources/quality-control-view.css]: updated

File: CoreConversationView/src/au/gov/asd/tac/constellation/views/conversationview/resources/conversation.css [hunk 5]
```diff
@@ -8,7 +8,7 @@
 
 
 .text-area .scroll-bar {
-    -fx-skin: ""; /* com.sun.javafx.scene.control.skin.ScrollBarSkin"; */
+    -fx-skin: ""; /* javafx.scene.control.skin.ScrollBarSkin"; */
```
Comment (cygnus-x-1) [inline CoreConversationView/src/au/gov/asd/tac/constellation/views/conversationview/resources/conversation.css]: any idea why we're not using this one?

Comment (arcturus2) [reply CoreConversationView/src/au/gov/asd/tac/constellation/views/conversationview/resources/conversation.css]: I wondered the same thing but assumed it was causing a graphics issue. I plan to clean this all up in #308.

**Pull request merged**